### ISSUE TITLE
Failed when USING Clp in Julia 1.0.0

### ISSUE THREAD

**Issue by hurak** [JuliaOpt/Clp.jl#31]
I fail when USING Clp in Julia 1.0.0 (on Linux)

```
julia> using Clp
[ Info: Precompiling Clp [e2554f3b-3117-50c0-817c-e040a3ddf72d]
ERROR: LoadError: LoadError: syntax: extra token "ClpModel" after end of expression
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include at ./sysimg.jl:29 [inlined]
 [4] include(::String) at /home/hurak/.julia/packages/Clp/cTlVN/src/Clp.jl:7
 [5] top-level scope at none:0
 [6] include at ./boot.jl:317 [inlined]
 [7] include_relative(::Module, ::String) at ./loading.jl:1038
 [8] include(::Module, ::String) at ./sysimg.jl:29
 [9] top-level scope at none:2
 [10] eval at ./boot.jl:319 [inlined]
 [11] eval(::Expr) at ./client.jl:389
 [12] top-level scope at ./none:3
in expression starting at /home/hurak/.julia/packages/Clp/cTlVN/src/ClpCInterface.jl:214
in expression starting at /home/hurak/.julia/packages/Clp/cTlVN/src/Clp.jl:10
ERROR: Failed to precompile Clp [e2554f3b-3117-50c0-817c-e040a3ddf72d] to /home/hurak/.julia/compiled/v1.0/Clp/UsKiE.ji.
Stacktrace:
 [1] error(::String) at ./error.jl:33
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] compilecache(::Base.PkgId, ::String) at ./loading.jl:1184
 [4] macro expansion at ./logging.jl:311 [inlined]
 [5] _require(::Base.PkgId) at ./loading.jl:941
 [6] require(::Base.PkgId) at ./loading.jl:852
 [7] macro expansion at ./logging.jl:311 [inlined]
 [8] require(::Module, ::Symbol) at ./loading.jl:834
 [9] eval at ./boot.jl:319 [inlined]
 [10] #53 at /home/hurak/.julia/packages/Atom/jJn7Y/src/repl.jl:125 [inlined]
 [11] with_logstate(::getfield(Main, Symbol("##53#55")), ::Base.CoreLogging.LogState) at ./logging.jl:397
 [12] with_logger(::Function, ::Atom.Progress.JunoProgressLogger) at ./logging.jl:493
 [13] top-level scope at /home/hurak/.julia/packages/Atom/jJn7Y/src/repl.jl:124
```

**Comment by odow** [comment]
Clp has not been updated for Julia v1.0. PR's accepted :)

**Comment by juliohm** [comment]
This is causing LightGraphsFlows.jl tests to fail, and hence it is holding updates downstream. Could you please take a look at this when you have time?

**Comment by odow** [comment]
If you need Clp on v1 urgently, PR's are accepted. I don't know if this is a priority for any of the JuMP developers.

Related: @juan-pablo-vielma has been working on using BinaryBuilder for Clp:
https://github.com/juan-pablo-vielma/ClpBuilder
https://github.com/juan-pablo-vielma/Clp.jl

**Comment by juan-pablo-vielma** [comment]
Is LightGraphsFlows.jl using pure Clp.jl or also JuMP.jl? Would it be hard to move to GLPK.jl? that seems to be passing at least v0.7 even without BB

**Comment by mlubin** [comment]
Clp is working on 1.0 and is tagged.

**Comment by markaduol** [comment]
No its not. I get the following error when attempting to add `Clp` on Julia v1.0.3.
```
  Updating registry at `C:\Users\maduo\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package LinQuadOptInterface [f8899e07]:
 LinQuadOptInterface [f8899e07] log:
 ├─possible versions are: [0.1.0, 0.2.0, 0.3.0-0.3.1, 0.4.0-0.4.3, 0.5.0, 0.6.0] or uninstalled
 ├─restricted by julia compatibility requirements to versions: [0.3.0-0.3.1, 0.4.0-0.4.3, 0.5.0, 0.6.0] or uninstalled
 ├─restricted by compatibility requirements with Gurobi [2e9cd046] to versions: [0.3.0-0.3.1, 0.4.0-0.4.3]
 │ └─Gurobi [2e9cd046] log:
 │   ├─possible versions are: [0.1.4-0.1.31, 0.2.0-0.2.8, 0.3.0-0.3.3, 0.4.0-0.4.1, 0.5.0-0.5.7] or uninstalled
 │   ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.4-0.1.31, 0.2.0-0.2.8, 0.3.0-0.3.3, 0.4.0-0.4.1, 0.5.0-0.5.7]
 │   └─restricted by julia compatibility requirements to versions: 0.5.0-0.5.7 or uninstalled, leaving only versions: 0.5.0-0.5.7
 └─restricted by compatibility requirements with MathOptInterface [b8f27783] to versions: 0.6.0 or uninstalled — no versions left
   └─MathOptInterface [b8f27783] log:
     ├─possible versions are: [0.1.0, 0.2.0, 0.3.0, 0.4.0-0.4.1, 0.5.0-0.5.1, 0.6.0-0.6.4, 0.7.0, 0.8.0] or uninstalled
     └─restricted to versions 0.8.0-0.8 by JuMP [4076af6c], leaving only versions 0.8.0
       └─JuMP [4076af6c] log:
         ├─possible versions are: 0.18.5 or uninstalled
         └─JuMP [4076af6c] is fixed to version 0.18.5+
```

**Comment by blegat** [comment]
It needs to be updated to the latest version of LQOI
